README.md

```diff
@@ -46,83 +65,150 @@
   </ul>
 </details>
 
-<details>
-  <summary>Math magicians</summary>
-  
-</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Perfumes basic math calculations**
+- **Generates some quootes**
+- **Makes easier to calculate basic math operations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is the live Demo👇🏻.
+
+- [Live Demo Link](https://husush23.github.io/Math-magician/)
```

Comment:
- [ ] This link is leading to a blank live page, kindly check if you used the correct link and not a broken one